test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3546,6 +3546,48 @@ def forward(self, x):
         x = torch.randn(3, 3)
         self.run_test(MyReluModule(), x)
 
+    def test_clip_int(self):
+        """Clip <= opset 11 does not support non-floats (e.g. ints)
```

Comment:
Remove the doc strings for all these tests. They really don't add any useful info. The fact that the test is present means that we want to enforce this case works for all opset_versions, which is enough info for the reader.
How that is implemented shouldn't be documented here.